gcp/rakefiles/entrypoint.rake

```diff
@@ -17,18 +20,13 @@ if @project_type.nil?
   raise ArgumentError, "@project_type must be set"
 end
 
-@exekube_cmd = "docker-compose run --rm --service-ports xk"
-
-@dot_config_path = "../../.config/#{@env}"
-CLOBBER << @dot_config_path
-
 desc "Create cluster and deploy GPII components to it"
-task :default => :deploy
+task :default => :call_deploy
 
 task :set_vars do
+  FileUtils.mkdir_p @dot_config_path
```

Comment:
I think this will be fixed in my GPII-2996 branch where I refactored this but FYI I believe Rake people prefer `directory`.

Reply:
I am fine with any of them.

Reply:
Nevermind, rake's `directory` somehow does not work properly with `..` in dir name.

Reply:
That sounds weird -- `directory` might need to go outside a `task` block? But if that doesn't work, `mkdir_p` is fine.